src/SDKs/EventGrid/management/Management.EventGrid/Microsoft.Azure.Management.EventGrid.csproj

```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.EventGrid</PackageId>
     <Description>Provides developers with a library to create and manage all Azure EventGrid resources. </Description>
-    <Version>4.0.1-preview</Version>
+    <Version>5.0.0</Version>
```

Comment:
Are you sure you wish to publish a stable version right away, generally we recommend publishing a few preview versions before going stable, that way you have some leeway to implement fixes and such which wouldn't be a breaking change to customers.

Reply:
These features had been in preview for quite sometime. We had some of them introduced in 2018-09-15-preview and in 2019-02-01-preview. We are GA'ing them here in 2019-06-01. So, we are good to go.